Dockerfile

```diff
@@ -304,25 +304,23 @@ RUN pip install bleach && \
     pip install decorator && \
     pip install entrypoints && \
     pip install html5lib && \
-    # Latest version breaks nbconvert: https://github.com/ipython/ipykernel/issues/422
-    pip install ipykernel==5.1.1 && \
```

Comment:
nbconvert/papermill is still in use, sessionless doesn't fix that, should probably test that separately.

Reply:
I was able to "save & run all" fine: https://screenshot.googleplex.com/BpV75KH44ZbLLvc